src/drive/mobile/store/configureStore.js

```diff
@@ -1,63 +1,6 @@
 /* global __DEVELOPMENT__ */
-import { saveState } from './persistedState'
-import RavenMiddleWare from 'redux-raven-middleware'
-import { createLogger } from 'redux-logger'
-import { ANALYTICS_URL, getReporterConfiguration } from '../lib/reporter'
-import { compose, createStore, applyMiddleware } from 'redux'
-import {
-  shouldEnableTracking,
-  getTracker,
-  createTrackerMiddleware
-} from 'cozy-ui/react/helpers/tracker'
-import thunkMiddleware from 'redux-thunk'
-import eventTrackerMiddleware from 'drive/middlewares/EventTracker'
-import rootReducer from 'drive/mobile/reducers'
-
-const configureStore = (initialState = {}) => {
-  const loggerOptions = __DEVELOPMENT__
-    ? {}
-    : {
-        level: {
-          prevState: false,
-          nextState: false
-        }
-      }
-  const loggerMiddleware = createLogger(loggerOptions)
-  const ravenMiddleWare = RavenMiddleWare(
-    ANALYTICS_URL,
-    getReporterConfiguration()
-  )
-  const middlewares = [thunkMiddleware, ravenMiddleWare]
-  if (shouldEnableTracking() && getTracker()) {
-    middlewares.push(eventTrackerMiddleware)
-    middlewares.push(createTrackerMiddleware())
-  }
-  middlewares.push(loggerMiddleware)
-
-  // Enable Redux dev tools
-  const composeEnhancers =
-    (__DEVELOPMENT__ && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose
-
-  const store = createStore(
-    rootReducer,
-    initialState,
-    composeEnhancers(applyMiddleware(...middlewares))
-  )
-
-  store.subscribe(() =>
-    saveState({
-      settings: store.getState().settings,
-      mobile: {
-        settings: store.getState().mobile.settings,
-        mediaBackup: {
-          uploaded: store.getState().mobile.mediaBackup.uploaded
-        }
-      },
-      availableOffline: store.getState().availableOffline
-    })
-  )
-
-  return store
-}
+const configureStore = __DEVELOPMENT__
+  ? require('./configureStore.dev').default
+  : require('./configureStore.prod').default
```

Comment:
Meh. Having 2 files for the store makes it very hard to spot the differences, but requires updating 2 files if we change the store. I think it's worth it if the 2 configs are very different, but I don't think they are?
Pinging in @goldoraf since I believe he had a strong opinion on this too.

Reply:
Ok, I shall remove this commit.